### DISCUSSION TITLE
(vim) Default Path for installation should be an expected standard (not c:\tools)

### DISCUSSION THREAD

**Discussion by gsmitheidw** [chocolatey-community/chocolatey-packages#1944] [Ideas]
(vim) installing to c:\tools is not expected and not something may be noticed without visiting the specific webpage. 
Could this be changed to **$env:ProgramFiles** instead?  This would follow a proper Windows standard path.
I understand there may be some who still run x86, so **${env:ProgramFiles(x86)}** may be needed too.

**Comment by AdmiringWorm** [comment]
The vim package makes use of an archive during installation.
Packages that pretty much extracts an archive instead of running an installer should not be installed into the Program Files directory.

As well as, if I remember correctly, the location that vim gets installed requires write access to the directory, which it would not have in the Program Files.

However, you can still override the installation directory by using the package parameter `InstallDir` (see https://docs.chocolatey.org/en-us/guides/create/parse-packageparameters-argument#installing-with-package-parameters on how to pass in package parameters)

**Comment by gsmitheidw** [comment]
Many thanks, that's fair enough - it seems a few problems with vim stem from upstream bad design issues from the source installer. In short this is not actually a chocolatey issue, this is a vim issue. 

The bigger question is should a package be admitted where the installer doesn't conform to security standards creating a user writable folder in the root of a drive? I doubt it would it pass muster to make a deb or rpm package in Linux that didn't conform to security - there'd probably be some uproar. It should either not require write access and go into program files, or install to %appdata% like other user based installs (like VS Code/MS Teams/etc). I fundamentally disagree with userspace installs - very wasteful and difficult to manage on a multiuser system but that's another story!
